_episodes/07-integration.md

```diff
@@ -101,13 +104,14 @@ the expected result should be, regression tests look to the past for the
 expected behavior. The expected
 result is taken as what was previously computed for the same inputs.
 
-> ## The Past as Truth {.callout}
+> ## Further explanation
```

Comment:
Nope. Will revert all those you've suggested. I honestly don't recall making them.